src/odemis/driver/hamamatsurx.py

```diff
@@ -256,12 +252,17 @@ def _setBinning(self, value):
         """
         # ResolutionVA need tuple instead of list of format [2 x 2]
         binning = "%s x %s" % (value[0], value[1])
+
+        # If camera is acquiring, it is essential to stop cam first and then change binning.
+        # Currently, this only affects the Alignment tab, where camera is continuously acquiring.
+        self.parent.AcqStop()  # TODO check with HW!
         self.parent.CamParamSet("Setup", "Binning", binning)
+        self.parent.AcqStart("Live")
 
         prev_binning, self._binning = self._binning, value
 
         # adapt resolution
-        # TODO check if really necessary
+        # TODO check if really necessary - can be removed??
```

Comment:
I think you can drop the TODO. It's kind of necessary, to follow the same behaviour as the other camera drivers.